packages/react/README.md

```diff
@@ -55,6 +67,40 @@ import Button from '@massds/mayflower-react/dist/Button';
 For a more detailed guide and information on the components included in Mayflower React and their functionality, visit our [Mayflower React Storybook][react-storybook]. Click on the Info and Knobs tabs for component prop types, details and options.
 
 
+### To consume the typed components in your typescript project:
+1. Create a `mayflower.d.ts` in your project and declare module imports by mayflower-react:
+```javascript
+declare module "@massds/mayflower-assets";
+declare module "@massds/mayflower-assets/static/images/*";
+```
+2. Import a mayflower-react typed component. For examples:
+```javascript
+import BrandBanner from "@massds/mayflower-react/dist/BrandBanner";
```

Comment:
how does the project know which index file to import? does it need to specify it somewhere?

Reply:
Could we say that it's just `@massds/mayflower-react/dist/[ComponentName]` with the `[ComponentName]` taken from the React storybook?
![Selection_012](https://user-images.githubusercontent.com/74615308/207937225-9940f69e-e623-434a-98fe-d23701ef7074.png)